content/posts/cloudbees-cross-team-and-dev-sec-ops.md

```diff
@@ -0,0 +1,272 @@
+
+
+There are a few good CloudBees' blog posts and CloudBees' documentation on CloudBees' Cross Team Collaboration: 
+
+- [Cross Team Collaboration (Part 1)](https://www.cloudbees.com/blog/cross-team-collaboration-part-1)
+- [Cross Team Collaboration (Part 2)](https://www.cloudbees.com/blog/cross-team-collaboration-part-2)
+- [Cross Team Collaboration documentation](https://go.cloudbees.com/docs/cloudbees-core/cloud-admin-guide/cross-team-collaboration/)
+
+## DevSecOps
+[DevSecOps](https://tech.gsa.gov/guides/understanding_differences_agile_devsecops/) - the idea of shifting security left in your Continuous Delivery pipelines - is becoming a vital component of successful CD. DevSecOps is all about speeding up software delivery, while maintaining, or even improving, the level of security for delivered application code. However, even though you should be shifting automated security left - you still don't want it to impede developers trying to deliver software more quickly. CloudBees' Cross Team Collaboration feature is a perfect capability for automating security while at the same time getting out of the way of developers - improving the security and quality of your software delivery while minimizing the impact on delivery speed.
+
+## Use Case: Asynchronously Scan Container Images for Vulnerabilities and Compliance
+As containers become a more and more ubiquitous method for delivering your applications, ensuring that your container images don't have security vulnerabilities and/or organization specific security compliance issues is an important aspect of CD for containerized application delivery. However, scanning containers images isn't the fastest process in the world and you don't want to unnecessarily slow down developers trying to get stuff done. You also may not want to depend on individual development teams to configure and manage important securitys steps in their delivery pipelines.
+
+Cross Team Collaboration enables you to publish an event from a [Pipeline Shared Library](https://jenkins.io/doc/book/pipeline/shared-libraries/) for [securely building container images](https://github.com/cloudbees-days/pipeline-library/blob/master/vars/kanikoBuildPush.groovy) and then asynchronously triggering *not-so-quick* security related jobs listening for events, making it very easy to provide security as part of the CD pipelines for an entire organization.
+
```

Comment:
*In terms of adding some more context into the use case...*

So, this jobs can be run in different teams or Jenkins masters that may require specific configurations, being able to run automatically thanks to the queue management from the events that is done by the Operations Center in [CloudBees Core](https://www.cloudbees.com/products/cloudbees-core). Pipelines for building containers and checking vulnerabilities are then decoupled but they run together any time you build a container (e.g. Every time you need to build a container the vulnerabilities will be checked, but running vulnerabilities check doesn't need to build the container)